packages/apps-config/src/endpoints/testing.ts

```diff
@@ -231,6 +231,13 @@ export function createTesting (t: TFunction): LinkOption[] {
       providers: {
         ZERO: 'wss://alphaville-0.zero.io'
       }
+    },
+    {
+      info: 'sgc',
+      text: t('rpc.sgc', 'Sgc', { ns: 'apps-config' }),
```

Comment:
Also adjust alphabetically.

Reply:
OK the latest change is third add sgc network Thank you very much!!!